src/PagesBody/UserDogsPage/UserDogPage.js

```diff
@@ -0,0 +1,44 @@
+import ClassOrDogButton from '../../Molecules/ClassOrDogButton/ClassOrDogButton';
+import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
+import MainButton from '../../Atoms/MainButton/MainButton';
+import { UserDataContext } from '../../Context/UserDataContext';
+import participants from '../../Data/MongoDBMock/participants';
+import { useContext } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+const UserDogPage = () => {
+  const navigate = useNavigate();
+
+  const { state } = useContext(UserDataContext);
+  console.log(state);
+  console.log(state.userId);
+  const participantDogs = participants.find(
+    (participant) => participant.participantId === state.userId,
+  ).dogs;
+  console.log(participantDogs);
+
+  return (
+    <ColumnWrapper paddingLeftRight={1} paddingTop={0.5}>
+      {participantDogs.map((dog, index) => {
+        const { dogName, dogId } = dog;
+        return (
+          <ClassOrDogButton
+            key={dogId}
+            dogInfo={{
+              index,
+              dogName,
```

Comment:
pass here `dogId` too
our ClassOrDogButton use this props:
![image](https://user-images.githubusercontent.com/75560322/152647096-b3a89f3d-77b6-49c1-8868-2443895be13a.png)

Reply:
I've updated a prop, but still struggling with this context usage. Maybe I'm trying to use a wrong file?